assets/js/index.js

```diff
@@ -415,6 +415,13 @@ const CostApi = {
   }
 }
 
+/**
+ * Allows functionality on collapsible lists 
+ * */
+$(document).ready(function(){
+  $('.collapsible').collapsible();
+});
+
 /**
```

Comment:
Please put this at the top of the file where the rest of the components are initialized, line 13 or so ```
$(document).ready(function(){
   ...
   $('.collapsible').collapsible()
   ...
})```